core/src/backends/eclair/rest/types.rs

```diff
@@ -268,3 +267,77 @@ impl TryInto<PayInvoiceResult> for PayInvoiceResponse {
         Ok(result)
     }
 }
+
+#[derive(Debug, Serialize)]
+pub struct DecodeInvoiceRequest {
+    invoice: String,
+}
+
+impl From<String> for DecodeInvoiceRequest {
+    fn from(invoice: String) -> DecodeInvoiceRequest {
+        return DecodeInvoiceRequest { invoice };
+    }
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
```suggestion
#[derive(Debug, Deserialize)]
```
I think we only need those for now.